package.json

```
@@ -22,20 +22,26 @@
   ],
   "scripts": {
     "clean": "rm -rf lib",
+    "build": "npm run compile",
     "compile": "tsc -b && echo Compiled",
     "depcheck": "depcheck",
     "docs": "npm run clean && npm run compile && oclif-dev readme",
-    "lint": "npm run lint:dev -- --quiet",
-    "lint:dev": "sort-package-json && eslint . --ext .ts --fix",
+    "lint": "eslint . --ext .ts --fix",
```

```suggestion
    "lint": "sort-package-json && eslint . --ext .ts --fix",
```
This should not be removed.